doc/onlinehelp.rst

```diff
@@ -1,29 +1,92 @@
-:orphan:
-
 ###########
 Online Help
 ###########
 
 The **Online Help** pane provides a built-in web browser to explore dynamically generated Python documentation on installed modules—including your own—rendered by a ``pydoc`` server running in the background.
 
-.. image:: images/online_help/online_help_standard.png
-   :width: 570px
-   :alt: Spyder Online Help panel on the index page, a list of builtin modules
+.. image:: images/online_help/online-help-standard.png
+   :alt: Spyder Online Help pane on the index page, a list of builtin modules
 
 |
 
 
+
 =====================
 Using the Online Help
 =====================
 
-Enter the name of the item you'd like documentation on in the :guilabel:`Module or package` field, or the :guilabel:`Get` field, to load its doc directly.
-Or, if you're not sure of the name of the object you want help on, or are looking for a specific keyword, use the :guilabel:`Search` field to get a list of results.
-You can view and re-run previous searches with the drop-down menu or the :kbd:`Down Arrow` key in the :guilabel:`Module or package field`, navigate previous pages with the forward and back buttons (top left, left and right arrows), and go back to the homepage with the home button (house icon top left) or the ``Module Index`` link top right.
+The Online Help's home shows an index where you can browse the documentation of standard library modules, third party packages installed in your Python environment, and your own local code.
+Click on the name of any module to open its documentation. 
+
+.. image:: images/online_help/online-help-browse.gif
+   :alt: Spyder Online Help pane showing module browsing
+
+|
+
+Enter the name of the item you'd like help on in the :guilabel:`Package` field to load its documentation directly.
+
+.. image:: images/online_help/online-help-name.gif
+   :alt: Spyder Online Help pane showing module browsing by name
+
+|
+
+The module's file location is linked to the right of the doc's title, which you can click to view its source code.
+Standard library modules also have a link to the corresponding Python.org docs, which can be viewed right inside of the pane.
+
+.. image:: images/online_help/online-help-path.gif
+   :alt: Spyder Online Help pane showing source code clicking module's path
+
+|
+
+If you're not sure of the name of the object you want help on, or are looking for a specific keyword, use the :guilabel:`Search` field to get a list of results.
+
+.. image:: images/online_help/online-help-search.gif
+   :alt: Spyder Online Help pane showing search of a keyword
+
+|
+
+Links above the search bar provide an index of topics with general help and a list of Python keywords linked to their corresponding docs.
+
+
+
+=============
+Toolbar items
+=============
+
+Just like in a web browser, the forward and back buttons (top left, left and right arrows) move through the pages you have visited, and the home button (house icon, top left) returns you to the module index.
```

Comment:
```suggestion
Just like in a web browser, the forward and back buttons (left and right arrows on top) move through the pages you have visited, and the home button (house icon) returns you to the module index.
```

I think there's no need to be so specific about the placement of those icons given that

* They are only ones of its kind in the pane.
* They are quite easy to spot in the gif below.